tests/models_core/test_server_logprobs.py

```diff
@@ -69,7 +69,7 @@ async def my_chat(
 
 @pytest.mark.parametrize("model", MODELS)
 @pytest.mark.parametrize("max_tokens", [32])
-@pytest.mark.parametrize("num_logprobs", [3])
+@pytest.mark.parametrize("num_logprobs", [5])
```

Comment:
yeah :) 

i'm not sure how many other places we need to set this to 5, but if there are other locations maybe we should turn this into a variable.